### DISCUSSION TITLE
What should I work on next for this repo?

### DISCUSSION THREAD

**Discussion by MoritzHayden** [MoritzHayden/wordle-prophecy#2] [Ideas]
Just curious.

**Comment by CalamityAdam** [comment]
Great question.. You should build something!

**Reply by MoritzHayden** [reply]
Great idea!

**Reply by isaiahcolson** [reply]
You should do something fun.

**Reply by isaiahcolson** [reply]
> Great question.. You should build something!

Noice

**Reply by MoritzHayden** [reply]
This is very helpful!

**Comment by isaiahcolson** [comment]
Why don't you try to rebuild the wordle game using typescript?

**Reply by CalamityAdam** [reply]
Helpful suggestion!

**Reply by MoritzHayden** [reply]
Thank you for the help! It is helpful.
